src/components/RadioButton/RadioButton.jsx

```diff
@@ -27,6 +27,7 @@ class RadioButton extends React.Component {
             name={this.props.name}
             onChange={this.props.onChange}
             checked={this.props.checked}
+            value={this.props.val}
```

Comment:
why `val` and not `value`?

Reply:
If you try to use this component with `redux-form`, it appears that the `value` props is overwritten by the `Field` component. (Not sure if there is a better way to pass in the value to the `input`).